src/cli/README_BR.md

```diff
@@ -180,10 +182,24 @@ Usage: `br prefixtable`
 
 Get the discovered prefixes by Border Routing Manager on the infrastructure link.
 
+Info per prefix entry:
+
+- The prefix
+- Whether or not the prefix is on-link (on-link or route)
```

Comment:
Do we need the parenthetical here?

Whether or not the prefix is on-link or route

Reply:
Updated in new push. Thanks.